Change active directory variable names for AWS

Edwin-Pau opened the pull request:
For the AWS single connector deployment, all service_account variables
have been renamed to ad_service_account to avoid confusion with other
types of service accounts.

Signed-off-by: Edwin Pau <epau@teradici.com>

Commit:
Change active directory variable names for AWS

For the AWS single connector deployment, all service_account variables
have been renamed to ad_service_account to avoid confusion with other
types of service accounts.

Signed-off-by: Edwin Pau <epau@teradici.com>

shermanyin commented on modules/aws/cac/cac-startup.sh.tmpl:
```diff
@@ -28,7 +28,7 @@ get_credentials() {
 
         PCOIP_REGISTRATION_CODE=$(aws kms decrypt --region ${aws_region} --ciphertext-blob fileb://<(echo "${pcoip_registration_code}" | base64 -d) --output text --query Plaintext | base64 -d)
-        SERVICE_ACCOUNT_PASSWORD=$(aws kms decrypt --region ${aws_region} --ciphertext-blob fileb://<(echo "${service_account_password}" | base64 -d) --output text --query Plaintext | base64 -d)
+        SERVICE_ACCOUNT_PASSWORD=$(aws kms decrypt --region ${aws_region} --ciphertext-blob fileb://<(echo "${ad_service_account_password}" | base64 -d) --output text --query Plaintext | base64 -d)
```
might as well change this too for consistency?

shermanyin commented on modules/aws/centos-gfx/centos-gfx-startup.sh.tmpl:
```diff
@@ -20,13 +20,13 @@ get_credentials() {
 
         PCOIP_REGISTRATION_CODE=${pcoip_registration_code}
-        SERVICE_ACCOUNT_PASSWORD=${service_account_password}
+        SERVICE_ACCOUNT_PASSWORD=${ad_service_account_password}
```
same here

shermanyin commented on modules/aws/centos-std/centos-std-startup.sh.tmpl:
```diff
@@ -20,13 +20,13 @@ get_credentials() {
 
         PCOIP_REGISTRATION_CODE=${pcoip_registration_code}
-        SERVICE_ACCOUNT_PASSWORD=${service_account_password}
+        SERVICE_ACCOUNT_PASSWORD=${ad_service_account_password}
```
here as well and same for other files

Edwin-Pau commented:
- Updated global variable names as requested and force pushed to update pull request.
- First time using rebase squash (had some issues with git when I rebased off staging).
- Changes tested and deployed successfully on AWS.
